README.md

```diff
@@ -27,6 +27,35 @@ How can you efficiently handle the resources of your server, knowing that you ar
 
 Enter Fastify. Fastify is a web framework highly focused on providing the best developer experience with the least overhead and a powerful plugin architecture. It is inspired by Hapi and Express and as far as we know, it is one of the fastest web frameworks in town.
 
+### Quick start
+
+Good tools make API's development quicker and easier to maintain that if you did everything by hand.
```

Comment:
=> "Good tools make API development quicker and easier to maintain than doing everything manually.

Reply:
done.